src/errors.js

```diff
@@ -82,9 +82,15 @@ export function onlyErrorsWithCode(
   return (error) => {
     let throwError = true;
 
-    if (Array.isArray(codeWanted) && codeWanted.indexOf(error.code) !== -1) {
-      throwError = false;
-    } else if (error.code === codeWanted) {
+    if (Array.isArray(codeWanted)) {
+      if (codeWanted.indexOf(error.code) !== -1) {
+        throwError = false;
+      }
+      if (codeWanted.indexOf(error.errno) !== -1) {
```

Comment:
You could _possibly_ even use [`Array#includes()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes), which I think is supported in Firefox 43+.